docs/user-guide.md

```diff
@@ -5,8 +5,17 @@ This is the user guide for Open-Local.
 ## Requirements
 
 - Kubernetes v1.18+
-- helm v3.0+
+- Helm v3.0+
```

Comment:
also include Linux distro we tested, and critical rpms like lvm2

Reply:
done